Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -3940,7 +3940,7 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         case CSSPropertyWebkitCursorVisibility:
 #endif
         case CSSPropertyWebkitFontSizeDelta:
-        case CSSPropertyWebkitHyphenateCharacter:
+        case CSSPropertyHyphenateCharacter:
```

Comment:
Separate bug, but we should support discrete animations on hyphenate-character

Reply:
I see. I will address that with a separate bug.